app/models/rules/basic.rb

```diff
@@ -1,4 +1,4 @@
-class Rules::Basic < Struct.new(:auction)
+class Rules::Basic < Rules::BaseRules
```

Comment:
Yes, I guess we should call it Reverse or Descending although neither option is particularly exciting